src/pages/studyView/StudyViewPage.tsx

```diff
@@ -194,10 +210,21 @@ export default class StudyViewPage extends React.Component<
             false,
             true
         );
+
+        this.toolbarLeftUpdater = setInterval(() => {
```

Comment:
what's dis?

Reply:
thats how i figure out how big to make the msk tabs. it keeps track of the leftmost coordinate of the study view toolbars

Reply:
would it make any sense for this to be handled inside MSKTabs?  isn't it just getting position of itself?  or perhaps it gets passed a function as a prop which returns the element when invoked?

Reply:
I feel like that's making MSKTabs too complicated for an unusual use case. I like just passing `width` to MSKTabs and handling that outside.